pkg/packagekit/autoupdates.jsx

```diff
@@ -359,6 +359,11 @@ export class AutoUpdates extends React.Component {
 
         return (<>
             <div id="autoupdates-settings">
+                {enabled && this.state.backend.parsing_failed &&
+                <Alert isInline
+                       variant="danger"
+                       className="autoupdates-card-error"
+                       title={_("Failed to parse unit files for dnf-automatic.timer or dnf-automatic-install.timer. Please remove custom overrides to enable this feature.")} />}
```

Comment:
Rather technical, but accurate and helpful for someone who knows what this is about. I first thought "let's add a link to the services page", but (1) that can't arbitrarily edit overrides either, and (2) we don't precisely know here *which* unit. So let's keep it as-is.